modin/pandas/test/test_groupby.py

```diff
@@ -439,6 +439,14 @@ def maybe_get_columns(df, by):
     ):
         # Not yet supported for non-original-column-from-dataframe Series in by:
         eval___getattr__(modin_groupby, pandas_groupby, "col3")
+        eval__getitem__(modin_groupby, pandas_groupby, "col3")
+
+    non_by_cols = (
```

Comment:
cases of intersection of 'by' columns and 'selection' is not supported yet

Reply:
I suggest adding this comment directly to the code and them removing it after support for cases of the intersection of 'by' columns and 'selection'.

Reply:
added